lib/cloud_controller/constants.rb

```diff
@@ -1,5 +1,5 @@
 module VCAP::CloudController
   class Constants
-    API_VERSION = '2.25.0'.freeze
+    API_VERSION = '2.25.1'.freeze
```

Comment:
@shalako will ask this version to be bumped in a separate story after all the service key work is complete. Please leave it as `2.25.0` (but update the `API_FOLDER_CHECKSUM`. Thanks!